### PR TITLE
[23282] - Add contributing file

### PR DESCRIPTION
**Pull request by catalinaalcala**
## Ticket
- [23282]

## Description
- Added CONTRIBUTING.md section and file to README
- Added CODE_OF_CONDUCT.md file
- Added License section to README

## Commit message
-  docs: add contributing.md [23282]

## Testing
- [ ] Added unit tests
- [ ] Ran unit tests successfully
- [x] Added documentation for public APIs and/or Wiki

### CODE REVIEW

**Comment by AuthyApps** [comment]
## Code coverage
### Total coverage: **`95.18%`**
> Powered by [Slather](https://github.com/SlatherOrg/slather)

<p align="right" data-meta="generated_by_UnitTests">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: README.md [hunk 1]
```diff
@@ -252,3 +254,14 @@ let twilioVerify = try builder.build()
 ```
 
 The push token will change after the reinstall. Update the push token to receive push notifications for challenges, as is explained in [Update factor's push token](#UpdatePushToken)
+
+<a name='Contributing'></a>
+
+## Contributing
+This project wolcomes contributions. Please check out our [Contributing guide](./CONTRIBUTING.md) to learn more on how to get started.
```
Comment (mejiagarcia) [inline README.md:261]: Here is a typo, should be `welcomes`

Comment (catalinaalcala) [reply README.md]: Fixed it, thank you.

**Comment by AuthyApps** [comment]
## Code coverage
### Total coverage: **`95.18%`**
> Powered by [Slather](https://github.com/SlatherOrg/slather)

<p align="right" data-meta="generated_by_UnitTests">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

**Review by mejiagarcia — APPROVED** [review]

**Review by yafuquen — COMMENTED** [review]
@catalinaalcala Can you remove this file https://github.com/twilio/twilio-verify-ios/blob/main/.github/CODE_OF_CONDUCT.md ? The idea will be to use the one you created

**Review by yafuquen — APPROVED** [review]

**Comment by AuthyApps** [comment]
## Code coverage
### Total coverage: **`95.18%`**
> Powered by [Slather](https://github.com/SlatherOrg/slather)

<p align="right" data-meta="generated_by_UnitTests">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

**Pull request merged**